pkg/config/config.go

```diff
@@ -54,12 +65,45 @@ type (
 )
 
 // FindByName find a table by its name.
-func (t Tables) FindByName(name string) (*Table, error) {
+func (t Tables) FindByName(name string) *Table {
 	for _, table := range t {
 		if table.Name == name {
-			return table, nil
+			return table
 		}
 	}
 
-	return nil, errors.New("table not found")
+	return nil
+}
+
+// LoadSpecFromFile loads klepto spec from file
+func LoadSpecFromFile(configPath string) (*Spec, error) {
+	if configPath != "" {
+		// Use config file from the flag.
+		log.Debugf("Reading config from %s ...", configPath)
+		viper.SetConfigFile(configPath)
+	} else {
+		log.Debugf("Reading config from %s ...", DefaultConfigFileName)
+
+		cwd, err := os.Getwd()
+		if err != nil {
+			return nil, wErrors.Wrap(err, "can't find current working directory")
+		}
+
+		viper.SetConfigName(".klepto")
+		viper.AddConfigPath(cwd)
+		viper.AddConfigPath(".")
```

Comment:
Why not use `viper.SetConfigFile` here as well?

Reply:
I'll remove this part in the next PRs completely as with default value set in flags there is no way to get here. But now I just left the function as I got in in another file.

Reply:
removed